homeassistant/components/mqtt/vacuum/schema_legacy.py

```diff
@@ -173,14 +173,14 @@ class MqttVacuum(MqttEntity, VacuumEntity):
 
     def __init__(self, hass, config, config_entry, discovery_data):
         """Initialize the vacuum."""
-        self._cleaning = False
+        self._attr_is_on = False
+        self._attr_battery_level = 0
+        self._attr_fan_speed = "unknown"
```

Comment:
Can we group them all, and sort them?

```suggestion
        self._attr_battery_level = 0
        self._attr_fan_speed = "unknown"
        self._attr_is_on = False
        self._attr_status = "Unknown"
```